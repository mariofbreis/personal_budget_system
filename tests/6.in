i 100 0 10 r Mesada
b 500
m 200
b 900
i 9000 900 0 d Mota
w 10 200
m 300
w 10 200
m 400
w 10 200
m 500
w 10 200
m 1000
w 10 200
