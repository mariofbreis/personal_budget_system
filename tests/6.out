5100
9100
930
940
950
960
Incomportavel
